src/components/sideMenu/AdminSideNav.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { History, PeopleAlt, Category } from '@material-ui/icons';
+
+import RouteNames from '../../routes/RouteNames';
+import { StyledLink, StyledIcon } from './style';
+const AdminSideNav = () => {
+  const { orderHistory, users, vendors, categories } = RouteNames;
+
+  return (
+    <>
+      <StyledLink to={orderHistory}>
```

Comment:
Can't we make a general Link component and used that in all side navs Instead of repeating ourself again and again ??

Reply:
It's only being used in SideNav.

Reply:
Yeah but it's repetitive so we should make a common component and used that every where